tmt.spec

```diff
@@ -93,6 +96,18 @@ Recommends: qemu-system-x86-core
 %description provision-virtual
 Dependencies required to run tests in a local virtual machine.
 
+%if 0%{?rhel} >= 9 || 0%{?fedora}
+%package provision-beaker-mrack
```

Comment:
Should we call the package just `tmt-provision-beaker`? In a similar way as we have `tmt-provision-virtual` and not `tmt-provision-virtual-testcloud`? @lukaszachy, @thrix, @happz, what do you think?

Reply:
`tmt-provision-beaker` sounds good to me.

Reply:
renamed